dspace-server-webapp/src/main/java/org/dspace/app/rest/model/HarvestedCollectionRest.java

```diff
@@ -0,0 +1,170 @@
+    private String harvestMessage;
+
+    @JsonProperty("metadata_config_id")
+    private String metadataConfigId;
+
+    @JsonProperty("harvest_status")
+    private HarvestStatusEnum harvestStatus;
+
+    @JsonProperty("harvest_start_time")
+    private Date harvestStartTime;
+
+    @JsonProperty("last_harvested")
+    private Date lastHarvested;
+
+    private HarvesterMetadataRest metadata_configs;
+
+    private CollectionRest collectionRest;
+
+    @JsonIgnore
+    @Override
+    public Integer getId() {
+        return id;
+    }
+
+    public String getCategory() {
+        return CATEGORY;
+    }
+
+    public Class getController() {
+        return HarvestedCollectionRest.class;
+    }
+
+    @JsonIgnore
+    public String getType() {
+        return NAME;
+    }
+
+    @JsonIgnore
+    public CollectionRest getCollectionRest() {
+        return this.collectionRest;
+    }
+
+    public void setCollection(CollectionRest collectionRest) {
+        this.collectionRest = collectionRest;
+    }
+
+    @JsonIgnore
+    public int getHarvestType() {
+        return harvestType.ordinal();
+    }
+
+    @JsonGetter("harvest_type")
+    public String getHarvestTypeAsString() {
+        return harvestType.name();
+    }
+
+    public void setHarvestType(HarvestTypeEnum harvestType) {
+        this.harvestType = harvestType;
+    }
+
+    public String getOaiSource() {
+        return oaiSource;
+    }
+
+    public void setOaiSource(String oaiSource) {
+        this.oaiSource = oaiSource;
+    }
+
+    public String getOaiSetId() {
+        return oaiSetId;
+    }
+
+    public void setOaiSetId(String oaiSetId) {
+        this.oaiSetId = oaiSetId;
+    }
+
+    public String getMetadataConfigId() {
+        return metadataConfigId;
+    }
+
+    public void setMetadataConfigId(String metadataConfigId) {
+        this.metadataConfigId = metadataConfigId;
+    }
+
+
+    public String getHarvestMessage() {
+        return harvestMessage;
+    }
+
+    public void setHarvestMessage(String harvestMessage) {
+        this.harvestMessage = harvestMessage;
+    }
+
+    @JsonIgnore
+    public HarvestStatusEnum getHarvestStatus() {
+        return harvestStatus;
+    }
+
+    @JsonGetter("harvest_status")
+    public String getHarvestStatusAsString() {
+        return harvestStatus == null ? null : harvestStatus.name();
+    }
+
+    public void setHarvestStatus(HarvestStatusEnum harvestStatus) {
+        this.harvestStatus = harvestStatus;
+    }
+
+    public Date getHarvestStartTime() {
+        return harvestStartTime;
+    }
+
+    public void setHarvestStartTime(Date harvestStartTime) {
+        this.harvestStartTime = harvestStartTime;
+    }
+
+    public Date getLastHarvested() {
+        return lastHarvested;
+    }
+
+    public void setLastHarvested(Date lastHarvested) {
+        this.lastHarvested = lastHarvested;
+    }
+
+    @LinkRest(linkClass = HarvesterMetadataRest.class, name = "metadata_configs", optional = true)
```

Comment:
I think this is also wrong.  I believe this should be `name=harvestermetadata` as well.  To clarify, the REST Contract now uses the terminology "harvestermetadata" (which is the same terminology in the existing DSpace Documentation and also the REST endpoint name: https://github.com/DSpace/Rest7Contract/blob/master/harvestermetadata.md). So we should *not* be returning anything named `metadata_configs`.   This looks like it might be the last instance of `metadata_configs` (beyond some awkwardly named variables, like the one in this method), but I haven't been able to test this PR yet to verify.